Added Danske Bank SE DABASESX (#29)
openpaymentseurope/docs: docs/banks/DABASESX.md

---
id: dabasesx
title: Danske Bank (DABASESX)

> sidebar_label: Danske Bank

Reviewer: Should this be Danske Bank Sweden?

Author: Well, it is listed in the Swedish section and I looked at their naming on their Swedish pages and looked like they only use "Danske Bank".. Bolagsverket says: "Danske Bank A/S, Danmark, Sverige Filial"

Reviewer: Cool, keep it as it is then :)

---

## Status Highlights

| Status | Product | Comment |
|:---|---|---|

> |![](https://img.shields.io/badge/status-important-important.svg)| PIS | Sandbox doesn't support specifying debtor account for payment, this is therefore omitted by us in the request to the ASPSP. The PSU must choose which account that should be debited in the ASPSP's SCA flow. |

Reviewer: Might not be needed to be so specific, could it be enough to say that the PSU must select the account that should be debited regardless of what is sent in the Payment Initiation request in sandbox?

|![](https://img.shields.io/badge/status-important-important.svg)| AIS | "Read Transaction Details" not supported by ASPSP. |

## Supported SCA Methods
|Environment     |SCA Method | Authentication Method | Status | Comment |
|----------------|----------|--------------|--------------|--------------|
|Sandbox         |OAuth Redirect | User/password credentials  | ![](https://img.shields.io/badge/status-active-success.svg) | Please see `Sandbox Test Data` |
|Production      |OAuth Redirect | Mobilt BankID | ![](https://img.shields.io/badge/status-active-success.svg) | When choosing debtor account for payment in the banks SCA webpages, a warning message "kontonummerverifikation misslyckades" is presented by the bank. It is unknown what this means and it has been reported to the bank, but does not seem to have any impact on the payment authorisation. |

### Sandbox Test Data
Credentials for SCA:
- Username: 8195475386
- Password: xUKSWPgHy2H2XBt8cv

> Sandbox only supports British accounts, this applies both to AIS and PIS, currency is therefore `GBP` and IBANs are for GB as well. We are mapping UK.OBIE.SortCodeAccountNumber to BBAN in sandbox to make it more aligned with the rest of the system.

Reviewer: ```suggestion
Sandbox only supports British accounts, this applies both to AIS and PIS, currency is therefore `GBP` and IBANs are for GB as well. We are mapping UK.OBIE.SortCodeAccountNumber to BBAN in sandbox to make it more aligned with other Swedish ASPSPs.
```
